pkgs/development/haskell-modules/configuration-ghc-7.10.x.nix

```diff
@@ -42,6 +42,10 @@ self: super: {
   # Build jailbreak-cabal with the latest version of Cabal.
   jailbreak-cabal = super.jailbreak-cabal.override { Cabal = self.Cabal_1_24_2_0; };
 
+  gtk2hs-buildtools = super.gtk2hs-buildtools.override { Cabal = self.Cabal_1_24_2_0; };
```

Comment:
Ah, okay. Sorry, I misread your patch, actually, and misunderstood the change. Thank you for the explanation.